checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -61,6 +62,47 @@ public CalledMethodsTransfer(CalledMethodsAnalysis analysis) {
         ((CalledMethodsAnnotatedTypeFactory) atypeFactory).calledMethodsValueElement;
   }
 
+  /**
+   * The Called Methods Checker is one of the sub-checkers of the Resource Leak Checker. When we
+   * enable -Ainfer for the Resource Leak Checker, whole-program inference (wpi) is performed for
+   * all the sub-checkers. However, our mechanism for inferring annotations for the resource leak
+   * checker is different. It relies on pattern matching instead. The wpi results for this inference
+   * are not useful and only end up slowing down the convergence of the algorithm for Resource Leak
+   * Inference.
+   *
+   * @param tree a tree
+   * @return false if Resource Leak Checker is running as one of the upstream checkers, otherwise
+   *     returns the result of the super call
+   */
+  @Override
+  protected boolean shouldPerformWholeProgramInference(Tree tree) {
+    if (isResourceLeakCheckerEnabled()) {
+      return false;
+    }
+    return super.shouldPerformWholeProgramInference(tree);
+  }
+
+  /**
+   * The Called Methods Checker is one of the sub-checkers of the Resource Leak Checker. When we
```

Comment:
Nit: avoid saying "we" in comments describing the behavior of a system: it is the system that does things, not "us" (the system's authors).

Reply:
also applies to "our" later in the comment